ad_block_client.cc

```diff
@@ -1420,6 +1451,14 @@ int serializeFilters(char * buffer, size_t bufferSizeAvail,
     }
     bufferSize++;
 
+    if (f->ruleDefinition != nullptr) {
+      if (buffer) {
+        snprintf(buffer + bufferSize, bufferSizeAvail, "%s", f->ruleDefinition);
+      }
+      bufferSize += static_cast<int>(strlen(f->ruleDefinition));
```

Comment:
any time serialization deserialization is changed, we need to update the adblock version, so I'm ok with this but you'd have to set it to version 5.